api/src/pcapi/core/mails/transactional/pro/venue_needs_picture.py

```diff
@@ -0,0 +1,22 @@
+from pcapi.core import mails
+from pcapi.core.mails import models
+from pcapi.core.mails.transactional.sendinblue_template_ids import TransactionalEmail
+from pcapi.core.offerers import models as offerer_models
+from pcapi.utils import urls
+
+
+def get_permanent_venue_needs_picture_data(venue: offerer_models.Venue) -> models.TransactionalEmailData:
+    return models.TransactionalEmailData(
+        template=TransactionalEmail.VENUE_NEEDS_PICTURE.value,
+        params={
+            "VENUE_NAME": venue.publicName or venue.name,
+            "VENUE_FORM_URL": urls.build_pc_pro_venue_link(venue),
+        },
+    )
+
+
+def send_permanent_venue_needs_picture(venue: offerer_models.Venue) -> bool:
+    data = get_permanent_venue_needs_picture_data(venue)
+    if venue.bookingEmail:
+        return mails.send(recipients=[venue.bookingEmail], data=data)
+    return False
```

Comment:
Les fonctions d'envoi de mail ont (normalement) le comportement suivant : elles ne renvoient `False` que si on a échoué à envoyer le mail (avec une erreur). Quand aucun mail n'est à envoyer, on renvoie `True`. Exemples : 

https://github.com/pass-culture/pass-culture-main/blob/60b24c719700b613fda9f471da9ffc0ad242de77/api/src/pcapi/core/mails/transactional/pro/first_venue_approved_offer_to_pro.py#L50-L51

https://github.com/pass-culture/pass-culture-main/blob/60b24c719700b613fda9f471da9ffc0ad242de77/api/src/pcapi/core/mails/transactional/pro/reminder_before_event_to_pro.py#L27-L28

Donc, ici, on devrait plutôt écrire : 

```suggestion
    if not venue.bookingEmail:
        return True
    return mails.send(recipients=[venue.bookingEmail], data=data)
```

Reply:
Bien vu ! Danke